pkg/tsdb/cloudwatch/response_parser_test.go

```diff
@@ -391,7 +391,7 @@ func Test_buildDataFrames_parse_label_to_name_and_labels(t *testing.T) {
 			},
 			Statistic:        "Average",
 			Period:           60,
-			MetricQueryType:  models.MetricQueryTypeQuery,
+			MetricQueryType:  models.MetricQueryTypeSearch,
```

Comment:
Were these set to the wrong query type before? Otherwise it seems like we're losing tests?

Reply:
Yea, they were the wrong query type before. `MetricQueryTypeSearch` uses dimensions, whereas `MetricQueryTypeQuery` doesn't use that field from the query object. It uses the `SqlExpression` field which is a SQL string that includes how the metrics are filtered and grouped.